### PR TITLE
suppressed filter_adj_plot_mat + added save_dir in compute_and_save_s…

### PR DESCRIPTION
**Pull request by davidmeunier79**
…pectral_connectivity, compute_and_save_multi_spectral_connectivity

and plot_circular_connectivity for test in tmp_dir

### CODE REVIEW

File: ephypype/spectral.py [hunk 1]
```diff
@@ -3,13 +3,15 @@
...
 import os
-
 import numpy as np
+
 from scipy.io import savemat
 
 from mne.connectivity import spectral_connectivity
 from mne.viz import circular_layout, plot_connectivity_circle
 
+import matplotlib.pyplot as plt
```
Comment (jasmainak) [inline ephypype/spectral.py]: can you make the matplotlib import nested? Otherwise it will result in weird bugs ...

Comment (davidmeunier79) [reply ephypype/spectral.py]: > can you make the matplotlib import nested? Otherwise it will result in weird bugs ...

done

**Comment by jasmainak** [comment]
@annapasca could you look at the diff and merge if you're happy? I don't know the code well enough to comment more ...

**Comment by jasmainak** [comment]
@dmalt could you add `externals/click` to the ignore list in `.coveragerc` ? We don't our coverage stats to go down because of that ...

**Comment by dmalt** [comment]
sure

**Comment by annapasca** [comment]
Thks @davidmeunier79 @jasmainak !

**Comment by dmalt** [comment]
#59

**Pull request merged**